.github/workflows/vulnerability_scan_subworkflow.yml

```diff
@@ -82,40 +37,49 @@ jobs:
       - name: Install xmllint
         uses: ./.github/actions/install-xmllint
 
-      - name: Get EE dist ZIP
+      - name: Generate ${{ matrix.image.label }} dist ZIP
         run: |
-          . .github/scripts/ee-build.functions.sh
-          HZ_VERSION=$(awk -F '=' '/^ARG HZ_VERSION=/ {print $2}' hazelcast-enterprise/Dockerfile)          
-          HAZELCAST_EE_ZIP_URL=$(get_hz_dist_zip "" "${HZ_VERSION}")        
-          curl --fail --silent --show-error --location "$HAZELCAST_EE_ZIP_URL" --output hazelcast-enterprise/hazelcast-enterprise-distribution.zip;
+          # Make a dummy empty ZIP file to avoid scanning Java dependencies, as managed downstream
```

Comment:
Could we use the real zip but remove all *.jar files from it? This way we would be a bit closer to the docker image we actually push. 

I have concerns about executable scripts from the distribution zips, if they're used in a safe manner, although I'm not sure if they're scanned. 

WDYT @kwart ?

Reply:
> Could we use the real zip but remove all *.jar files from it? This way we would be a bit closer to the docker image we actually push.
> 
> I have concerns about executable scripts from the distribution zips, if they're used in a safe manner, although I'm not sure if they're scanned.
> 
> WDYT @kwart ?

I'm _assuming_ that the distribution ZIPs are already scanned and considered _safe_, so further tests are redundant/duplicated.

Reply:
Jars are scanned for sure, not sure about shell scripts. I don't have strong opinion here. 
Don't know if it makes any difference, if you compare the scan results it might be an indicator it it makes sense